interface/patient_file/report/patient_report.php

```diff
@@ -198,8 +311,8 @@ function show_date_fun(){
    <input type='checkbox' name='include_employer' id='include_employer' value="employer"><?php xl('Employer','e'); ?><br>
    -->
    <input type='checkbox' name='include_insurance' id='include_insurance' value="insurance"><?php xl('Insurance','e'); ?><br>
-   <input type='checkbox' name='include_billing' id='include_billing' value="billing"
-    <?php if (!$GLOBALS['simplified_demographics']) echo 'checked'; ?>><?php xl('Billing','e'); ?><br>
+   <label id="opt_fee"><input type='checkbox' name='include_billing' id='include_billing' value="billing"
+    <?php if (!$GLOBALS['simplified_demographics']) echo 'checked'; ?>><?php xl('Billing','e'); ?><br></label>
```

Comment:
Converting a script to the new security model and making extensive changes in the same PR actually makes it more difficult to review, which is why it's best to separate them.